tex/gregoriotex-main.tex

```diff
@@ -1537,7 +1488,7 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 \def\grechangestyle#1#2{%
-  \IfStrEq{#1}{biginitial}{\gre@deprecated{biginitial style}{initial style}}{}%%% DEPRECATED for removal in 5.0
+  \IfStrEq{#1}{biginitial}{\gre@obsolete{biginitial style}{initial style}}{}%%% DEPRECATED for removal in 5.0
```

Comment:
This is not obsolete yet.  The deprecation of biginitial style was introduced in develop.